avocado/core/html.py

```diff
@@ -273,7 +242,7 @@ def _copy_static_resources(self):
                     shutil.copy(source, dest)
 
     def _render_report(self):
```

Comment:
Apart from this one it looks good and I think it can be also cherry_picked (when the last reference to result_dict is removed)